### PR TITLE
Ghostbusters: Don't send empty job applications to Slack

### PR DESCRIPTION
**Pull request by iain-macdonald**
If there's something strange
In your Slack channel
Who you gonna call
Ghostbusters!

An invisible candidate
And we can't reach them
Who you gonna call
Ghostbusters!

I ain't hirin' no ghost
I ain't hirin' no ghost

If you're seeing Slacks
And they are empty
Who can you call?
(Ghostbusters)

An invisible candidate
Lookin' for a job
Ow, who you gonna call?
(Ghostbusters)

**Related issues**: N/A

### CODE REVIEW

**Review by jdhollen — APPROVED** [review]

File: website/src/components/apply/apply.tsx [hunk 1]
```diff
@@ -35,6 +35,19 @@ function Component() {
...
+  if (
+    form.firstName.current.value.trim() == "" &&
+    form.lastName.current.value.trim() == "" &&
+    form.email.current.value.trim() == "" &&
+    form.phone.current.value.trim() == "" &&
+    form.linkedin.current.value.trim() == "" &&
+    form.github.current.value.trim() == "" &&
+    form.blurb.current.value.trim() == ""
+  ) {
+    alert("Please fill out the form before submitting");
```
Comment (siggisim) [inline website/src/components/apply/apply.tsx:47]: Alert!? Idk if I'd want to work at a place that uses alert() unironically

Comment (tylerwilliams) [reply website/src/components/apply/apply.tsx:47]: this is ironic usage

Comment (jdhollen) [reply website/src/components/apply/apply.tsx:47]: :(

Comment (siggisim) [reply website/src/components/apply/apply.tsx:47]: If you want to add a TODO for me to use an alert service type thing to show a butter bar or something like we do in the app, that's cool with me.

**Review by jdhollen — APPROVED** [review]

File: website/src/components/apply/apply.tsx (hunk 1 above)
Comment (iain-macdonald) [reply website/src/components/apply/apply.tsx]: looking forward to your beautiful alert-thats-not-an-alert

**Pull request merged**